build_resolvers/lib/src/build_asset_uri_resolver.dart

```diff
@@ -125,7 +125,7 @@ class BuildAssetUriResolver extends UriResolver {
         return _AssetState(path, _cachedAssetDependencies[id]!);
       }
       if (isChange) {
-        resourceProvider.updateFile(path, content);
+        resourceProvider.modifyFile(path, content);
```

Comment:
sure the pkg:analyzer version we have has this?

Reply:
yes